test/unit/blocks_music.js

```diff
@@ -1,31 +1,32 @@
 const test = require('tap').test;
```

Comment:
Should this file be renamed `test/unit/extension_music.js`?

Reply:
done!